paddle/fluid/operators/optimizers/adam_op.cc

```diff
@@ -236,6 +236,10 @@ class AdamWOpMaker : public AdamOpMaker {
  public:
   void Make() {
     AdamOpMaker::Make();
+    AddAttr<float>("lr_ratio",
```

Comment:
why add this argument to adam? is that adamw and adam share the same .cc file ?

in this case, adamw should have its own .cc file

Reply:
AdamWOpMaker inherits AdamOpMaker, and they use the same InferShape function of AdamOp.
In this case, 'lr_ratio' has no effect on Adam.